fem/fespace.hpp

```diff
@@ -748,6 +749,13 @@ class FiniteElementSpace
 
    int GetConformingVSize() const { return vdim * GetNConformingDofs(); }
 
+   /// Return the dimension of an vector in the space, including if elements
+   /// are vector-valued
```

Comment:
This could use a bit more explanation. If I understand correctly, the finite elements themselves may be vector-valued (e.g. $H(\mathrm{div})$ or $`H(\mathrm{curl})`$), say with $n$ components, and the finite element space may have a "vdim", i.e. we may be on the space $[V_h]^m$ which is like $m$ copies of $V_h$. `GetVectorDim()` will return in this case $n \times m$. Is that right?

Maybe this can be spelled out explicitly and mention in the docs why this is different from `FiniteElementSpace::GetVDim`. Maybe the documentation for `FiniteElementSpace::GetVDim` should also be adjusted.

Reply:
See https://github.com/mfem/mfem/pull/4598#issuecomment-2524254283